### ISSUE TITLE
question about color bars

### ISSUE THREAD

**Issue by jdbarnes-isi** [cpanato/mattermost-plugin-alertmanager#11]
I noticed that color is set dynamically, but being unfamiliar with go code and classing, I wasn't sure if it is possible to operate on the "severity" label to set a color of #ffcc00 for warnings instead of red.

I will play with the code later if I have time.

**Comment by cpanato** [comment]
We can update to use the name instead of hex code

**Comment by jdbarnes-isi** [comment]
Right, the issue isnt the name, but the ability to set a color based on some other string (like severity, or other arbitrary value) to set colors on the attachment... For example, alertmanager can send Warning and Critical, but right now all show red bars, whether it's a critical issue or not.

I will definitely be looking at this when I get through some go tutorials & documentation, unless you get to it first.

**Comment by cpanato** [comment]
@jdbarnes-isi ahh got it! I will see how I can use that, thanks for the insight

**Comment by cpanato** [comment]
I was taking a look at the code, and today it set the color bar based on the Alertmanager Status field, which can be "firing" (red) or  "resolved" (green)

the fields you mentioned are more of a severity field, and this usually is used to route the alert to a specific endpoint.

the severity field already shows up in the message. The severity is more tricky because you can set your own severity name
